docker-compose.yml

```diff
@@ -181,6 +181,21 @@ services:
       ports:
         - "127.0.0.1:8001:80"
 
+    pygoat:
+        build:
+            context: tests/appsec/integrations/pygoat_tests
+            dockerfile: Dockerfile.pygoat.2.0.1
+        ports:
+            - "127.0.0.1:8000:8000"
+        environment:
+            - DD_APPSEC_ENABLED=true
+            - DD_IAST_ENABLED=true
+            - DD_IAST_REQUEST_SAMPLING=100
+            - DD_IAST_VULNERABILITIES_PER_REQUEST=100
+            - DD_REMOTE_CONFIGURATION_ENABLED=true
+            - DD_AGENT_PORT=8126
+            - DD_TRACE_AGENT_URL=http://testagent:8126
```

Comment:
Are `DD_AGENT_PORT` and `DD_TRACE_AGENT` both necessary?

Reply:
Seem like its not, thanks!